TheaterSchedule.DAL/Repositories/PerfomanceRepository.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Linq.Expressions;
+using Entities.Models;
+using TheaterSchedule.DAL.Interfaces;
+
+namespace TheaterSchedule.DAL.Repositories
+{
+    public class PerfomanceRepository: IPerfomanceRepository 
+    {
+        TheaterDatabaseContext db;
+        public PerfomanceRepository(TheaterDatabaseContext db)
+        {
+            this.db = db;
+        }
+       
+        public List<Performance> GetPerformanceTitles(int settingsId)
+        {
+            int languageId = db.Settings.Where(a => a.SettingsId == settingsId).Select(c => c.LanguageId).First();
+            var definedPerformances = db.Performance.Include(a => a.PerformanceTr);

```

Comment:
query with includes will load all translations of performance
This is because Include load tree structure of objects
use joins to get flat object with only needed translation